tests/testthat/test-jjcs_num_formats.R

```diff
@@ -29,90 +29,94 @@ test_that("jjcs_num_formats works", {
 
   # jjcsformat_xx_SAS/R format tests
   expect_snapshot({
-    format_value(values, format = jjcsformat_xx_SAS("xx.x (xx.xx)"))
-    format_value(values, format = jjcsformat_xx_R("xx.x (xx.xx)"))
-    format_value(c(5.05, values[2]), format = jjcsformat_xx_SAS("xx.x (xx.xx)"))
-    format_value(c(5.05, values[2]), format = jjcsformat_xx_R("xx.x (xx.xx)"))
-    format_value(c(5.15, values[2]), format = jjcsformat_xx_R("xx.x (xx.xx)"))
+    format_value(values, format = jjcsformat_xx("xx.x (xx.xx)"), round_type = "sas")
```

Comment:
For all of these tests we need to use number(s) that give different values for the different types of rounding. otherwise we can't tell if its working or not.